use-cases/virgo/data.py

```diff
@@ -42,98 +51,97 @@ def execute(self) -> pd.DataFrame:
             pd.DataFrame: dataset of Q-plot images.
         """
         df_aux_ts = generate_dataset_aux_channels(
-            1000, 3, duration=16, sample_rate=500,
-            num_waves_range=(20, 25), noise_amplitude=0.6
+            1000,
+            3,
+            duration=16,
+            sample_rate=500,
+            num_waves_range=(20, 25),
+            noise_amplitude=0.6,
         )
         df_main_ts = generate_dataset_main_channel(
             df_aux_ts, weights=None, noise_amplitude=0.1
         )
 
         # save datasets
-        save_name_main = 'TimeSeries_dataset_synthetic_main.pkl'
-        save_name_aux = 'TimeSeries_dataset_synthetic_aux.pkl'
+        save_name_main = "TimeSeries_dataset_synthetic_main.pkl"
+        save_name_aux = "TimeSeries_dataset_synthetic_aux.pkl"
         df_main_ts.to_pickle(os.path.join(self.data_root, save_name_main))
         df_aux_ts.to_pickle(os.path.join(self.data_root, save_name_aux))
 
         # Transform to images and save to disk
         df_ts = pd.concat([df_main_ts, df_aux_ts], axis=1)
         df = generate_cut_image_dataset(
-            df_ts, list(df_ts.columns),
-            num_processes=20, square_size=64
+            df_ts, list(df_ts.columns), num_processes=20, square_size=64
         )
-        save_name = 'Image_dataset_synthetic_64x64.pkl'
+        save_name = "Image_dataset_synthetic_64x64.pkl"
         df.to_pickle(os.path.join(self.data_root, save_name))
         return df
 
 
-class SyntheticTimeSeriesDataset(Dataset):
-    def __init__(self, root_folder: Optional[str] = None,):
+class SyntheticTimeSeriesDatasetHDF5(Dataset):
+    def __init__(
+        self,
+        hdf5_file_location: str,
+        chunk_size: int = 500,
```

Comment:
I don't really understand the purpose of the chunk_size argument, what is that needed for?

Reply:
Ah okay, then it's what I thought initially and pretty intuitive, I think it doesn't need explaining then :)